tests/test_operators.py

```diff
@@ -39,15 +39,22 @@
                 }
             },
         ],
-        True,
+        False,
     ),
 )
 operator_contains = (
     "pre.json",
     "operator",
     {"params": {"mode": "contains", "operator_data": "EVPN"}},
     "result[0].vrfs.default.peerList[*].[$peerAddress$,peerGroup]",
-    ([{"7.7.7.7": {"peerGroup": "EVPN-OVERLAY-SPINE"}}], False),
```

Comment:
I do not get this: here we want to get items that contain the word `EVPN`. The result here is what I would expect for `not-contains`